src/test/java/actions/Actions.java

```diff
@@ -9,5 +9,31 @@
 // (авторизация отдельно, операции с заказами отдельно,
 // регистрация пользователя отдельно и т.п.)
 
+import pojo.Film;
+import pojo.Pilot;
+import services.Service;
+
+import java.util.List;
+
 public class Actions {
```

Comment:
Could you please clarify some details about this class? Does it have a difference from service classes? What is the main target of this class and could we replace it using the service classes?